cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/clinical/CVRClinicalDataProcessor.java

```diff
@@ -54,12 +55,20 @@
     @Autowired
     public CvrSampleListUtil cvrSampleListUtil;
     
+    Logger log = Logger.getLogger(CVRClinicalDataProcessor.class);
+
     @Override
     public CompositeClinicalRecord process(CVRClinicalRecord i) throws Exception {
-        List<String> record = new ArrayList<>();
+	if (i == null) return null;
+	List<String> record = new ArrayList<>();
         List<String> seqDateRecord = new ArrayList<>();
         for (String field : CVRClinicalRecord.getFieldNames()) {
-            record.add(cvrUtilitiess.convertWhitespace(i.getClass().getMethod("get" + field).invoke(i).toString().trim()));
+            try {
+                record.add(cvrUtilitiess.convertWhitespace(i.getClass().getMethod("get" + field).invoke(i).toString().trim()));
+	    } catch (NullPointerException e) {
```

Comment:
looks like one final white spacing adjustment is needed here, the `}` should align with `try`